src/server/hset_family.cc

```diff
@@ -135,6 +139,14 @@ struct HMapWrap {
     VisitMut(ov);
   }
 
+  void Launder(tiering::SerializedMapDecoder* dec) {
+    Overloaded ov{
+        [](StringMap* s) {},
+        [&](detail::ListpackWrap& lw) { *dec->Write() = lw; },
```

Comment:
this expression looks confusing. Is there an option to improve the interface? specifically, I do not understand what this `Write()` method applies to.